griptape/structures/pipeline.py

```diff
@@ -11,52 +11,58 @@
 
 @define
 class Pipeline(Structure):
-    def first_task(self) -> Optional[BaseTask]:
-        return self.tasks[0] if self.tasks else None
+    def add_task(self, task: BaseTask) -> BaseTask:
```

Comment:
Add `@deprecated` [with a description](https://peps.python.org/pep-0702/) to this method.

Reply:
What do you think about keeping `add_task`? My reasoning is that `add_task` is a generic sounding method that makes sense for all Structures. Moving `append_task` to `Structure` feels awkward for structures like `Agent`, whereas `add_task` still fits. Then Structures like Pipeline and Workflow can add extra helper methods like `append_task` and `insert_task`.

Reply:
I think the only thing I dislike is that we are going to have two methods doing the same thing, which increases API ambiguity. I agree that `Agent.append_task` sounds awkward and is also ambiguous. I wonder now if we should just keep `app_task` and get rid of `append_task`.

Reply:
Good point, ok let's just remove `append_task`